newIDE/app/src/ResourcesEditor/Toolbar.js

```diff
@@ -5,6 +5,7 @@ import ToolbarIcon from '../UI/ToolbarIcon';
 import ToolbarSeparator from '../UI/ToolbarSeparator';
 
 type Props = {|
+  onOpenRessourcesFolder: () => void,
```

Comment:
Almost! onOpenResourcesFolder

Reply:
Also, you might as well call it onOpenProjectFolder - there is nothing specific to resources.